patches/chrome-tools-build-win-create_installer_archive.py.patch

```diff
@@ -1,33 +1,41 @@
 diff --git a/chrome/tools/build/win/create_installer_archive.py b/chrome/tools/build/win/create_installer_archive.py
-index ffbf9ec0727b9a81eb4b8488ae67b797ec81a437..cdd0f6769f6629b855a56f267724673a79fec836 100755
+index ffbf9ec0727b9a81eb4b8488ae67b797ec81a437..3e84c7328a1dd51bd162225947de13e946e85d8c 100755
 --- a/chrome/tools/build/win/create_installer_archive.py
 +++ b/chrome/tools/build/win/create_installer_archive.py
-@@ -20,7 +20,7 @@ import os
- import shutil
- import subprocess
- import sys
--
+@@ -42,6 +42,7 @@ SETUP_PATCH_FILE_PREFIX = "setup_patch"
+ TEMP_ARCHIVE_DIR = "temp_installer_archive"
+ VERSION_FILE = "VERSION"
+ 
 +from create_installer_archive_helper import SignAndCopyPreSignedBinaries, BraveCopyAllFilesToStagingDir
  
- ARCHIVE_DIR = "installer_archive"
+ g_archive_inputs = []
  
-@@ -112,6 +112,7 @@ def CopyAllFilesToStagingDir(config, distribution, staging_dir, build_dir,
+@@ -112,6 +113,7 @@ def CopyAllFilesToStagingDir(config, distribution, staging_dir, build_dir,
    if enable_hidpi == '1':
      CopySectionFilesToStagingDir(config, 'HIDPI', staging_dir, build_dir,
                                   verbose)
 +  BraveCopyAllFilesToStagingDir(config, staging_dir, g_archive_inputs)
  
    if include_snapshotblob == '1':
      CopySectionFilesToStagingDir(config, 'SNAPSHOTBLOB', staging_dir, build_dir,
-@@ -551,6 +552,7 @@ def main(options):
+@@ -179,7 +181,7 @@ def GetPrevVersion(build_dir, temp_dir, last_chrome_installer, output_name):
+                                    output_name + ARCHIVE_SUFFIX)
+   cmd = [lzma_exec,
+          'x',
+-         '-o"%s"' % temp_dir,
++         '-o./"%s"' % temp_dir,
```

Comment:
why is this change needed?

Reply:
Mentioned here - https://github.com/brave/brave-core/pull/5484#discussion_r422034439

Reply:
I think the problem here is what we're passing in for `last_chrome_installer` because we shouldn't have to change the script itself. We should be passing in a a rebased value here https://github.com/brave/brave-core/pull/5484/files#diff-769a945034c01d9bc9d33ecfa819c902R18

Reply:
sorry, I guess this is `staging_dir`, but in any case this should be running in the root build dir so there's no reason for it to end up in the root of the filesystem